broadway/event-store-mongodb/0.2/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "container": {
+        "mongodb_uri": "mongodb://127.0.0.1/",
+        "mongodb_database": "default"
```

Comment:
I assume these will be different on different environments. Lets put them as environment variables: https://github.com/symfony/recipes#env-configurator